lib/src/rules/directives_ordering.dart

```diff
@@ -83,10 +124,24 @@ class _Visitor extends SimpleAstVisitor {
 
   @override
   void visitCompilationUnit(CompilationUnit node) {
+    _checkDartImportGoFirst(node);
+    _checkPackageImportBeforeRelative(node);
+  }
+
+  void _checkDartImportGoFirst(CompilationUnit node) {
     node.directives
         .where(_isImportDirective)
         .skipWhile(_isDartImport)
         .where(_isDartImport)
         .forEach(rule._reportLintWithDartImportGoFirstMessage);
   }
+
+  void _checkPackageImportBeforeRelative(CompilationUnit node) {
+    node.directives
+        .where(_isImportDirective)
+        .where(_isNotDartImport)
+        .skipWhile(_isAbsoluteImport)
+        .where(_isPackageImport)
+        .forEach(rule._reportLintWithPackageImportBeforeRelativeMessage);
+  }
```

Comment:
It is true, but more generally, is the idea of which rule has more priority than others, and in that way, we think that the order in the guidelines is the better